tasks/main.yml

```diff
@@ -68,24 +73,20 @@
   when: timesync_mode == 2
 
 - name: Get chrony version
-  command: rpm -q --qf '%{version}' chrony
-  args:
-    warn: false
-  register: timesync_chrony_version
-  changed_when: false
-  check_mode: false
-  when: timesync_mode != 2 and timesync_ntp_provider == 'chrony'
+  set_fact:
```

Comment:
Please do not set a fact, use vars/main.yml and lazy evaulation, or set it on a block similarly to a (let) construct in LISP (which would have the advantage of making the variable really local to the block).

Reply:
Fixed.